{{cookiecutter.stack_name}}/.github/workflows/docker.yml

```diff
@@ -4,14 +4,27 @@ name: Build, test, and publish Docker Images
 # https://docs.github.com/en/actions/reference/events-that-trigger-workflows
 on:
   pull_request:
-    paths-ignore:
-      - "*.md"
+    paths:
+      - ".github/workflows/docker.yml"
+
+      - "image/**"
+
+      - "tests/**"
+      - "Makefile"
+      - "requirements-dev.txt"
   push:
     branches:
       - master
       - main
-    paths-ignore:
-      - "*.md"
+    paths:
+      - ".github/workflows/docker.yml"
```

Comment:
This is feature creep for the PR, but this is an example of a name that I think isn't as helpful as `"build-image.yaml"` or `"test-build-push-image.yaml"` or similar.

Reply:
I think it follows the name of the "upstream" project.
https://github.com/jupyter/docker-stacks/tree/master/.github/workflows

But I think your naming makes more sense and I will probably do it soon in separate PR(s).